src/client/runner/automation/playback/press/index.js

```diff
@@ -141,6 +153,18 @@ export default class PressAutomation {
     }
 
     run () {
+        var activeElement = domUtils.getActiveElement();
+        var isIframe      = domUtils.isIframeElement(activeElement);
```

Comment:
let's rename to something like `isElementInIframe`. Otherwise I read `isIframe` as `we are in the iframe context now`